src/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://unicons.iconscout.com/release/v4.0.8/css/line.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <script src="https://kit.fontawesome.com/1bc3a7e061.js" crossorigin="anonymous"></script>
+    <title>To do list</title>
+  </head>
+  <body>
+    <main class="initial-container">
+      <div class="sub-container">
+        <header>
+          <div class="display-div">
+            <h2>Today's To Do</h2>
+            <i id="refresh" class="fa fa-refresh"></i>
+          </div>
+        </header>
+        <hr class="hr" />
+        <form>
+          <input
+            type="text"
+            id="new-task-add"
+            placeholder="Add to your list..."
+          />
+          <button class="icon-uil" type="submit"><i class="uil uil-enter"></i></button>
+         
+        </form>
+        <div>
+          <ul id="todo-task-list"></ul>
+          <p id="clear" class="wipe-all-out">Clear all completed</p>
+        </div>
+      </div>
+    </main>
```

Comment:
- Optional
I would recommend fixing the design issue. As you can see, the three dots should be inside the container as per the design provided. Kindly fix this issue as it does not look good.
![image](https://github.com/Zuwairanajma/To-do-List/assets/33268927/2a3ebd44-c719-4a99-9ebd-911a34305c69)